Pass the genre name instead of the identifier to the id3 command as the
numbers are offset by one.

--- abcde-2.7.1/abcde.orig	2015-10-04 11:30:28.000000000 +0200
+++ abcde-2.7.1/abcde	2016-01-09 15:49:12.249326957 +0100
@@ -822,11 +822,11 @@
 			case "$ID3SYNTAX" in
 				id3)
 					run_command tagtrack-$OUTPUT-$1 nice $ENCNICE \
 						$TAGGER $TAGGEROPTS -c "$COMMENTOUTPUT" \
 						-A "$DALBUM" -a "$TRACKARTIST" -t "$TRACKNAME" \
-						-y "$CDYEAR" -g "$GENREID" \
+						-y "$CDYEAR" -g "$CDGENRE" \
 						-T "${TRACKNUM:-$1}" \
 						"$ABCDETEMPDIR/track$1.$OUTPUT"
 					;;
 				id3v2)
 					# FIXME # track numbers in mp3 come with 1/10, so we cannot
